packages/flutter_rearch/lib/src/side_effects/text_editing_controller.dart

```diff
@@ -4,7 +4,9 @@ TextEditingController _textEditingController(
   WidgetSideEffectRegistrar use, {
   String? initialText,
 }) {
-  final controller = use.memo(() => TextEditingController(text: initialText));
-  use.effect(() => controller.dispose, [controller]);
-  return controller;
+  return use.disposable(
+    () => TextEditingController(text: initialText),
+    (controller) => controller.dispose(),
+    [initialText],
```

Comment:
The controller should not be recreated if `initialText` changes, so would you mind removing the `[initialText],` line?

Reply:
Done

Reply:
TY